yarn-project/p2p/src/attestation_pool/memory_attestation_pool.test.ts

```diff
@@ -0,0 +1,66 @@
+import { type PrivateKeyAccount } from 'viem';
+
+import { InMemoryAttestationPool } from './memory_attestation_pool.js';
+import { generateAccount, mockAttestation } from './mocks.js';
+
+const NUMBER_OF_SIGNERS_PER_TEST = 4;
+
+describe('MemoryAttestationPool', () => {
+  let ap: InMemoryAttestationPool;
+  let signers: PrivateKeyAccount[];
+
+  beforeEach(() => {
+    ap = new InMemoryAttestationPool();
+    signers = new Array(NUMBER_OF_SIGNERS_PER_TEST).fill(0).map(() => generateAccount());
+  });
```

Comment:
Just a fun tip: you can
`signers = Array.from({ length: NUMBER_OF_SIGNERS_PER_TEST }, generateAccount);`